mod/metrics/set/all/colours.rb

```diff
@@ -35,12 +35,24 @@ def mono_colour
     ]
   end
 
+  # Colours for non-numbers
+  def light_colour
```

Comment:
Everywhere else including css it's `color` not `colour`

Reply:
Shouldn't we use British English within code :D